### ISSUE TITLE
Pie chart - Tooltip and contrast - Accessibility issue

### ISSUE THREAD

**Issue by JVKE93** [DFE-Digital/accessibility-issues-statements#46]
Issue 6: Reflow & Contrast Minimum - Error
Location: [Dashboard](https://accessibility-statements.education.gov.uk/dashboard)
Severity: Medium

When viewed at high magnification the WCAG count hover tooltip appears outside the viewport. The text ‘Other’ also fails contrast minimum.
For users viewing the service on different viewports or at high levels of magnification, the tooltip would not be visible or would require horizontal scrolling. For people with low vision or a visual impairment the ‘other’ text may be difficult to read.

Recommendation:
Ensure that the tooltip reflows or repositions to stay inside the viewport.
Change the ‘Other’ text to a foreground colour which contrast sufficiently with the grey background colour.

![Image](https://github.com/user-attachments/assets/feeaba16-f0aa-4a1c-9c4f-17270dc2300e)
